docs/api/cli-options.md

```diff
@@ -145,13 +170,75 @@ For example, `storybook@latest upgrade --dry-run` will perform a dry run (no act
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
+| `--help`                 | Output usage information <br/>`storybook automigrate --help`                                                                                                                                                         |
+| `-c`, `--config-dir`     | Directory where to load Storybook configurations from<br/>`storybook automigrate --config-dir .storybook`                                                                                                            |
+| `-n`, `--dry-run`        | Only checks for fixes, not actually executing them<br/>`storybook automigrate --dry-run`                                                                                                                             |
+| `-s`, `--skip-install`   | Skip installing dependencies whenever applicable<br/> `storybook automigrate --skip-install`                                                                                                                         |
+| `-y`, `--yes`            | Skips interactive prompts and automatically accepts automigrations<br/>`storybook automigrate --yes`                                                                                                                 |
+| `-l`, `--list`           | Shows a list of available automigrations<br/> `storybook automigrate --list`                                                                                                                                         |
+| `--package-manager`      | Sets the package manager to use when installing the addon.<br/> Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook automigrate --package-manager pnpm`                                      |     |
+| `--renderer`             | Sets the renderer package for the framework Storybook is using.<br/> Useful for monorepo settings where multiple Storybook renderers are present in the same package.json<br/>`storybook automigrate --renderer vue` |     |
+| `--debug`                | Outputs more logs in the CLI to assist debugging<br/>`storybook automigrate --debug`                                                                                                                                 |
+| `--disable-telemetry`    | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md)<br/>`storybook automigrate --disable-telemetry`                                                                                |
+| `--enable-crash-reports` | Enable sending crash reprots to Storybook's telemetry. Learn more about it [here](../configure/telemetry.md).<br/>`storybook automigrate --disable-telemetry`                                                        |
```

Comment:
When you have a moment, can you update this table to the following:
```suggestion
| Option                   | Description                                                                                                                                                                                                             |
| ------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `-c`, `--config-dir`     | Directory where to load Storybook configurations from<br/>`storybook upgrade --config-dir .storybook`                                                                                                                   |
| `-l`, `--list`           | Shows a list of available automigrations<br/> `storybook automigrate --list`                                                                                                                                            |
| `-n`, `--dry-run`        | Checks for available migrations without applying them<br/>`storybook automigrate --dry-run`                                                                                                                             |
| `-s`, `--skip-install`   | Skips the migration check step during the upgrade process<br/>`storybook upgrade --skip-check`                                                                                                                          |
| `-y`, `--yes`            | Applies available migrations automatically without prompting for confirmation<br/>`storybook upgrade --yes`                                                                                                             |
| `--help`                 | Output usage information <br/>`storybook automigrate --help`                                                                                                                                                            |
| `--package-manager`      | Sets the package manager to use when running the auto migration.<br/>Available package managers include `npm`, `yarn`, and `pnpm`<br/>`storybook automigrate --package-manager pnpm`                                    |
| `--renderer`             | Specifies Storybook's renderer to use when running the automigration.<br/> Useful for monorepo environments where multiple Storybook instances can exist in the same project<br/>`storybook automigrate --renderer vue` |
| `--debug`                | Outputs more logs in the CLI to assist debugging<br/>`storybook automigrate --debug`                                                                                                                                    |
| `--disable-telemetry`    | Disables Storybook's telemetry. Learn more about it [here](../configure/telemetry.md#how-to-opt-out).<br/>`storybook build --disable-telemetry`                                                                         |
| `--enable-crash-reports` | Enables sending crash reports to Storybook's telemetry. Learn more about it [here](../configure/telemetry.md#crash-reports-disabled-by-default)<br/>`storybook dev --enable-crash-reports`                              |
```

The rationale behind this is that if you check the preview, the table is not rendered correctly and already includes the changes mentioned in other places (e.g., `dev` command) for accuracy

Reply:
I applied the most text changes I could, but some things didn't make sense e.g. storybook build, storybook dev being used in the snippets (while this is for storybook automigrate)

Reply:
This was to demonstrate the pattern and wording. Appreciate you updating it so promptly 🙏